server/oneoff/sync_name.py

```diff
@@ -0,0 +1,118 @@
+import json
+import os
+from app import session_scope
+from app.database.model_item_translation import ModelItemTranslation
+from app.database.model_spell_translation import ModelSpellTranslation
+from oneoff.sync_item import allowed_file_names as item_file_names
+
+app_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+allowed_file_names = item_file_names + ["spells"]
+
+
+def update_name(db_session, file_name):
+    with open(
+        os.path.join(app_root, "app/database/data/{}.json".format(file_name)), "r"
+    ) as file:
+        data = json.load(file)
+        name_to_record_map = {}
+
+        if file_name in item_file_names:
+            for r in data:
+                name_to_record_map[r["name"]["en"]] = r
+        else:
+            for r in data:
+                for pair in r["spells"]:
+                    for spell in pair:
+                        name_to_record_map[spell["name"]["en"]] = spell
+
+        while True:
+            old_name = input(
+                "Enter the old name of the record (as it exists in the db) that you would like to update or 'q' to quit: "
+            )
+            if old_name == "q":
+                return
+            new_name = input(
+                "Enter the new name of the record (as it should appear) that you would like to update: "
+            )
+
+            translations = None
+            if file_name in item_file_names:
+                translations = (
+                    db_session.query(ModelItemTranslation)
+                    .filter(
+                        ModelItemTranslation.locale == "en",
+                        ModelItemTranslation.name == old_name,
+                    )
+                    .all()
+                )
+            else:
```

Comment:
Let's do elif file_name == "spells" and throw an exception otherwise to prevent bugs if we add more file names that this can be used with.